storage/src/mmr/verification.rs

```diff
@@ -26,7 +27,7 @@ pub trait Storage<H: CHasher> {
     fn size(&self) -> u64;
```

Comment:
Should we make this async for good measure?

Reply:
done